cfgov/cdntools/management/commands/purge_cache.py

```diff
@@ -0,0 +1,89 @@
+import sys
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from wagtail.contrib.frontend_cache.utils import PurgeBatch, get_backends
+
+
+class Command(BaseCommand):
+    help = "Perorm invalidation purges of the frontend cache"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--backend",
+            nargs="+",
+            help="Limit the purge the given frontend cache backend name",
+        )
+        parser.add_argument(
+            "--all",
+            action="store_true",
+            help="Purge all URLs for all backends. ",
+        )
+        parser.add_argument(
+            "--cache_tag",
```

Comment:
```suggestion
            "--cache-tag",
```

For consistency? This will automatically get changed to `cache_tag`, see argparse docs [here](https://docs.python.org/dev/library/argparse.html#dest).